include/aspect/particle/world.h

```diff
@@ -67,8 +69,55 @@ namespace aspect
 
     namespace internal
     {
+      // This class evaluates the solution vector at arbitrary positions inside a cell.
+      // This base class only provides the interface for SolutionEvaluatorsImplementation.
+      // See there for more details.
```

Comment:
Can you turn all the comments into doxygen comments (`/** .... */`)? That way they show up in the online docs. That was not needed when this was in the .cc file, because they are not included.